freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/keytab/doc/KeytabModelNotes.java

```diff
@@ -0,0 +1,6 @@
+package com.sequenceiq.freeipa.api.v1.keytab.doc;
+
+public class KeytabModelNotes {
+    public static final String GET_KEYTAB_NOTES = "Creates the host and priciapl in FreeIpa and then creates and returns the keytab";
+    public static final String DELETE_KEYTAB_NOTES = "Will not delete the host as the same host might be chared by other pricipals";
```

Comment:
This should probably state what is actually deleted. Do you think that make it clearer for users of the API? So maybe the API is split the API into 2 separate APIs:
1. Delete a kerberos principal
2. Delete a host (and associated kerberos principals)

This avoids the confusion about "why did deleting a keytab do other things like delete a principal"?

What do you think?

(same comment for the actual API and the DESCRIBE_DELETE_KEYTAB text)

Reply:
agree